pool/helpers.go

```diff
@@ -86,11 +103,32 @@ func executeJob(ctx context.Context, id string, j amboy.Job, q amboy.Queue) {
 
 	if err := j.Error(); err != nil {
 		grip.Error(message.WrapError(err, msg))
+		span.SetStatus(codes.Error, "job encountered error")
+		span.RecordError(err, trace.WithStackTrace(true))
 	} else {
 		grip.Info(msg)
 	}
 }
 
+func jobSpan(ctx context.Context, j amboy.Job) (context.Context, trace.Span) {
```

Comment:
Can we mention in a doc comment that `jobSpan` is a no-op for Amboy queues running in applications that don't have Honeycomb hooked up already?

Reply:
done